docs/pages/access-controls/reference.mdx

```diff
@@ -81,7 +81,7 @@ spec:
     logins: [root, '{{internal.logins}}']
     # Windows logins a user is allowed to use for desktop sessions.
     windows_desktop_logins: [Administrator, '{{internal.logins}}']
-    # If kubernetes integration is enabled, this setting configures which 
+    # If kubernetes integration is enabled, this setting configures which
```

Comment:
```suggestion
    # If the Kubernetes integration is enabled, this setting configures which
```
I know it's not related to your PR, but using this opportunity to clean up the grammar slightly.

Reply:
You actually made the same comment on a different PR, and I've fixed it there, so going to leave this one as is.

Reply:
Oops, thanks for noticing